presto-native-execution/scripts/setup-macos.sh

```diff
@@ -16,7 +16,7 @@ set -eufx -o pipefail
 # Run the velox setup script first.
 source "$(dirname "${BASH_SOURCE}")/../velox/scripts/setup-macos.sh"
 
-MACOS_DEPS="${MACOS_DEPS} bison gperf"
+MACOS_VELOX_DEPS="${MACOS_VELOX_DEPS} gperf"
```

Comment:
Do we need to include `MACOS_VELOX_DEPS`?
The sourcing of the velox setup-macos script would set this but is the idea that brew will install both velox and prestissimo deps. This would then alleviate the need to run the velox script separately (prior). I found that running only the prestissimo script will not work on the Linux because dnf or apt isn't run to install all the deps (e.g. compiler and such) when run for the first time.

Perhaps we should do this similarly for Linux to make sure such deps are installed?

Reply:
I fixed the macos script.
The new setup scripts by default do not install anything.
We source the Velox setup scripts and install the build deps, deps from package manager, and then the packages via download.

Reply:
> I found that running only the prestissimo script will not work on the Linux because dnf or apt isn't run to install all the deps

Can you try now?